packages/block-editor/src/components/grid/grid-visualizer.js

```diff
@@ -54,29 +54,21 @@ const GridVisualizerGrid = forwardRef(
 		const [ isDroppingAllowed, setIsDroppingAllowed ] = useState( false );
 
 		useEffect( () => {
-			const observers = [];
-			for ( const element of [ gridElement, ...gridElement.children ] ) {
-				const observer = new window.ResizeObserver( () => {
-					setGridInfo( getGridInfo( gridElement ) );
-				} );
-				observer.observe( element );
-				observers.push( observer );
-			}
```

Comment:
It’s unnecessary to create a new `ResizeObserver` instance for each element given the callback can be shared and here it can be.